akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -903,7 +903,6 @@ Host: example.com
 
   class TestSetup {
     val hostname = "localhost"
-    //val (hostname, port) = SocketUtil.temporaryServerHostnameAndPort()
```

Comment:
remove the line entirely?

Reply:
yes, this commented line was present in the code and I have removed it.

Reply:
oops, sorry, good change indeed :D